src/utils/auth.ts

```diff
@@ -1,29 +1,35 @@
-const userNameValidation = (userName: any, setuserNameInvalid: any) => {
+const userNameValidation = (userName: any) => {
+  let validation;
```

Comment:
```suggestion
  const validation = regEx.test(userName);
  
  return validation
```

regex test function return a boolen.

it's good practice we think in a way to reduce unnecessary code.